essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/ZoomRoom/ZoomRoom.cs

```diff
@@ -2578,11 +2578,12 @@ public void LinkZoomRoomToApi(BasicTriList trilist, ZoomRoomJoinMap joinMap)
 
 			PasswordRequired += (devices, args) =>
 			{
-                Debug.Console(0, this, "***********************************PaswordRequired. Message: {0} Cancelled: {1} Last Incorrect: {2} Failed: {3}", args.Message, args.LoginAttemptCancelled, args.LastAttemptWasIncorrect, args.LoginAttemptFailed);
+                Debug.Console(2, this, "***********************************PaswordRequired. Message: {0} Cancelled: {1} Last Incorrect: {2} Failed: {3}", args.Message, args.LoginAttemptCancelled, args.LastAttemptWasIncorrect, args.LoginAttemptFailed);
 
 				if (args.LoginAttemptCancelled)
 				{
 					trilist.SetBool(joinMap.MeetingPasswordRequired.JoinNumber, false);
+					_meetingPasswordRequired = false;

```

Comment:
We should remove this line.  It's already handled inside OnPasswordRequired helper method

Reply:
Removed as recommended.